src/react-apps/applications/ux-editor/src/containers/FormFiller.tsx

```diff
@@ -64,15 +64,23 @@ export class FormFillerComponent extends React.Component<IFormFillerProps, IForm
   public saveFormData = () => {
     const altinnWindow: IAltinnWindow = window as IAltinnWindow;
     const { reportee, org, service, instanceId } = altinnWindow;
+    let routePrefix: string = null;
+    if (window.location.origin.includes('altinn.studio') || window.location.origin.includes('altinn3.no')) {
+      routePrefix = '/runtime';
+    }
     FormFillerActionDispatchers.submitFormData(`
-        ${window.location.origin}/runtime/api/${reportee}/${org}/${service}/${instanceId}`);
+        ${window.location.origin}${routePrefix}/api/${reportee}/${org}/${service}/${instanceId}`);
   }
 
   public submitForm = () => {
     const altinnWindow: IAltinnWindow = window as IAltinnWindow;
     const { reportee, org, service, instanceId } = altinnWindow;
+    let routePrefix: string = null;
+    if (window.location.origin.includes('altinn.studio') || window.location.origin.includes('altinn3.no')) {
```

Comment:
Would it not be better to just have a chek for this at app initialize and store the value in redux store? What mode we are in? We have all this checks everywhere now.

Reply:
These checks will be removed when we have solved #2029. My personal experience is that to store something like this in the redux store implies extending interfaces, reducers and actions. My personal opinion is to not make more work out of this (since it's going to be removed), but it is much cleaner code, and not needed to duplicate code like this.

Reply:
Fair enough.